src/tasks/sep/050-op-betanet-add-game-type/README.md

```diff
@@ -0,0 +1,21 @@
+# 050-op-betanet-add-game-type
+
+Status: [READY TO SIGN]
+
+## Objective
+
+This task adds the dispute game type 0 (Permissionless) to the Dispute Game Factory on OP Betanet for U18.
+
+## Simulation & Signing
+
+Simulation commands for each safe:
+```bash
+cd src/tasks/sep/050-op-betanet-add-game-type
+SIMULATE_WITHOUT_LEDGER=1 SKIP_DECODE_AND_PRINT=1 just --dotenv-path $(pwd)/.env simulate
```

Comment:
Even though this is the first task, you can still use `just simulate-stack` / `just sign-stack` here too, we should always use those by default going forward